documentation/modules/metrics/kafka-bridge/proc_kafka-bridge-configuring.adoc

```diff
@@ -0,0 +1,53 @@
+// Module included in the following assemblies:
+//
+// metrics/assembly_metrics-kafka-bridge.adoc
+
+[id='proc-kafka-bridge-configuring-{context}']
+= Configuring Kafka Bridge
```

Comment:
Do we really need this procedure? Would it be enough to just mention somewhere that you enable metrics by setting `enableMetrics: true`?

Reply:
My though was following the Kafka Exporter documentation. We have a section for Kafka related metrics and for Kafka Exporter so I was thinking to add a related section for Kafka Bridge.

Reply:
Maybe we should mention the option in the using guide https://strimzi.io/docs/operators/master/using.html#assembly-deployment-configuration-kafka-bridge-str

Reply:
I don't mind it as a procedure. Readers get used to a certain format, it reassuring :-)  I think the next steps and ref to the schema is good too. Maybe we can trim it...

Reply:
I added a link to the assembly monitoring bridge from the bridge configuration section, in order to allow people to come here and see how to enable metrics for the bridge itself.